src/app/modules/main/wallet_section/collectibles/controller.nim

```diff
@@ -50,6 +63,21 @@ proc init*(self: Controller) =
     let args = OwnedCollectiblesUpdateArgs(e)
     self.refreshCollectibles(args.chainId, args.address)
 
+  self.events.on(SIGNAL_REFRESH_COLLECTIBLES) do(e:Args):
+    let args = RetryCollectibleArgs(e)
+    let chainId = self.networkService.getNetworkForCollectibles().chainId
+    for address in args.addresses:
+      self.resetOwnedCollectibles(chainId, address)
```

Comment:
the reset will emit signal `SIGNAL_OWNED_COLLECTIBLES_RESET`, which will trigger a refresh (see line 56), so the line below is not needed

Reply:
done!!